config/vufind/BrowZine.ini

```
@@ -53,6 +53,13 @@ ignored_images[] = "https://assets.thirdiron.com/default-journal-cover.png"
 ;filter[] = "browzineWebLink"
 ;filter[] = "fullTextFile"
 
+; When this setting is enabled, the BrowZine-supplied "recommendedLinkText" label will be
+; used for each given "best" link, and the label defined below is ignored. Default is false.
+; This applies only to the "bestIntegratorLink" service where recommended text is supplied.
+; Note that Third Iron may change this text over time, and VuFind will attempt localization,
+; its current strings are not defined in /languages files.
```

Would these added words make your intent more clear?

```suggestion
; but its current strings are not currently defined in the /languages files.
```